python/cudf/cudf/_lib/pylibcudf/filling.pyx

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from cython.operator cimport dereference
+from libcpp.memory cimport unique_ptr
+from libcpp.utility cimport move
+
+from cudf._lib.cpp.column.column cimport column
+from cudf._lib.cpp.filling cimport (
+    fill as cpp_fill,
+    fill_in_place as cpp_fill_in_place,
+    repeat as cpp_repeat,
+    sequence as cpp_sequence,
+)
+from cudf._lib.cpp.table.table cimport table
+from cudf._lib.cpp.types cimport size_type
+
+from .column cimport Column
+from .scalar cimport Scalar
+from .table cimport Table
+
+
+cpdef Column fill(
+    object destination,
+    size_type begin,
+    size_type end,
+    object value,
+):
+
+    """Fill destination column from begin to end with value.
+    ``destination ``must be a
+    :py:class:`~cudf._lib.pylibcudf.column.Column`. ``value`` must be a
+    :py:class:`~cudf._lib.pylibcudf.scalar.Scalar`.
```

Comment:
This is redundant, the types are in the doc. Also, leave a newline after the first doc line.